darwin/dataset/utils.py

```diff
@@ -667,3 +668,40 @@ def get_annotations(
             with annotation_path.open() as f:
                 record = json.load(f)
             yield record
+
+
+def load_pil_image(path: Path):
```

Comment:
It's not the first time I see this function (I reckon we use it in Gale). Again, let's make sure we don't replicate this code elsewhere, otherwise this function looks good

Reply:
It's exactly the same function we use in Gale. But since we're not going to have Gale as a dependency of `darwin-py` that's why I copied here. Does Gale have `darwin-py` as a dependency already?